src/manager/params.rs

```diff
@@ -0,0 +1,14 @@
+use cid::Cid;
+use fvm_ipld_encoding::RawBytes;
+use fvm_shared::MethodNum;
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+
+/// Init actor Exec Params, see https://github.com/filecoin-project/builtin-actors/blob/master/actors/init/src/types.rs#L17
+#[derive(Serialize_tuple, Deserialize_tuple, Debug)]
```

Comment:
This hasn't been moved yet to fvm-utils right?

Reply:
moved, resolving this